packages/trpc/server/routers/viewer/slots.ts

```diff
@@ -89,6 +99,62 @@ const checkIfIsAvailable = ({
   const slotEndTime = time.add(eventLength, "minutes").utc();
   const slotStartTime = time.utc();
 
+  const fixedDateOverrides: {
+    start: Date;
+    end: Date;
+  }[] = [];
+
+  //check if date override for slot exists
+  let dateOverrideExist = false;
+
+  if (
```

Comment:
If a date override exists on the day of the slot and the slot starts before that or ends after that date override it will return false (not available)